model/project_ref.go

```diff
@@ -510,6 +522,21 @@ func (t TriggerDefinition) Validate(parentProject string) error {
 	return nil
 }
 
+func (d PeriodicBuildDefinition) Validate() error {
```

Comment:
should this be a pointer receiver?

Reply:
Don't think it matters here but I changed it